apps/web/src/app/not-found.tsx

```diff
@@ -1,49 +1,29 @@
-"use client";
-
-import { useEffect } from "react";
+import Image from "next/image";
 import Link from "next/link";
-import { useRouter } from "next/navigation";
-
-export default function NotFound() {
-  const router = useRouter();
-
-  useEffect(() => {
-    const handleKeydown = (e: KeyboardEvent) => {
-      if (e.key.match(/^[a-zA-Z0-9 ]$/)) {
-        void router.push("/");
-      }
-    };
-
-    window.addEventListener("keydown", handleKeydown);
 
-    return () => {
-      window.removeEventListener("keydown", handleKeydown);
-    };
-  });
+import { Footer } from "@/components/footer";
+import { SiteHeader } from "@/components/site-header";
+import { Heading } from "@/components/typography/heading";
+import { Text } from "@/components/typography/text";
 
+export default function NotFound() {
   return (
-    <div className="min-h-screen bg-[#000088] font-block text-[#dddddd] selection:bg-gray-800/60">
-      <main className="mx-auto max-w-3xl space-y-10 px-4 py-20 text-xl md:text-3xl">
-        <h1 className="text-4xl md:text-6xl">Finner ikke siden du ser etter...</h1>
-
-        <div className="space-y-4">
-          <p>Oops, ser ut som noen har rotet med koden her!</p>
-
-          <p>
-            Vi leter etter den savnede siden, men finner den ikke. Kanskje det er et tegn på at du
-            bør komme deg ut og strekke på bena? Ta en kort spasertur og nyt noen frisk luft, mens
-            vi prøver å fikse dette!
-          </p>
-
-          <p>Trykk hvilken som helst tast for å gå tilbake til forsiden.</p>
-        </div>
-
-        <div className="text-center">
-          <Link href="/" className="p-4 hover:underline">
-            Klikk her for å gå tilbake til forsiden
-          </Link>
-        </div>
-      </main>
+    <div className="flex min-h-screen flex-col">
+      <SiteHeader />
+      <Heading className="text-center"> PAGE NOT FOUND</Heading>
```

Comment:
```suggestion
      <Heading className="text-center">PAGE NOT FOUND</Heading>
```

Jeg tenker kanskje det også kunne stå på norsk? Vet ikke hva andre tenker.